build.zig

```diff
@@ -4,16 +4,14 @@ pub fn build(b: *Build) void {
     const target = b.standardTargetOptions(.{});
     const optimize = b.standardOptimizeOption(.{});
 
-    _ = b.addModule("ansi_term", .{
-        .root_source_file = b.path("src/main.zig"),
-    });
-
-    var main_tests = b.addTest(.{
+    const module = b.addModule("ansi_term", .{
         .root_source_file = b.path("src/main.zig"),
         .target = target,
         .optimize = optimize,
     });
```

Comment:
As I'm not too familiar with zig modules in the wild, is it normal to specify `target` and `optimize` for the _module_? I would expect that packages that depend on us will specify their own `target` and `optimize` settings for our module.

Reply:
Ok, makes sense.